ci/gitlab-cscs.yml

```diff
@@ -87,6 +121,15 @@ multi node release:
     SLURM_NTASKS: 2
     ALLOCATION_NAME: arbor-ci-release-$CI_PIPELINE_ID
 
+test-ault-fujitsu:
```

Comment:
Minor nitpick: I think replacing the `-` with spaces would match the other test names better. Also the `_` in `notify_github_*`)

Reply:
I changed them all to underscore, because on balance that was the most widely used convention in the files.